agent/core/triage_graph.py

```
@@ -81,8 +81,21 @@ async def retrieve_history_node(
     clinic_sops = sop_item.value if sop_item else {"rules": []}
     
+    # --- TEMPORAL EXPIRY (30-day TTL) ---
+    # If the brain hasn't been consolidated in 30 days, ignore it.
+    updated_at_str = clinic_sops.get("updated_at")
+    if updated_at_str:
+        try:
+            updated_at = datetime.fromisoformat(updated_at_str.replace("Z", "+00:00"))
+            if datetime.now(timezone.utc) - updated_at > timedelta(days=30):
+                print(f"--- KNOWLEDGE STALE: Knowledge is >30 days old ({updated_at_str}). Ignoring. ---")
+                clinic_sops = {"rules": []}
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)